### PR TITLE
Feature/fa 210 update static pages

### PR DESCRIPTION
**Pull request by CH-JonathanBlake**
__Short description outlining key changes/additions__
Updated static page content to match most recent designs. Covers most non logged in content except from home controller.
__JIRA Ticket Number__
fa-210
### Type of change

* [ ] Bug fix
* [X] New feature
* [X] Breaking change

### Pull request checklist

* [X] I have added unit tests for new code that I have added
* [X] I have added/updated functional tests where appropriate
* [X] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

File: src/main/resources/templates/dev-hub/authentication.html [hunk 1]
```diff
@@ -10,57 +10,48 @@
...
-    application which can be sent with any
-    <code>GET</code>
-    request for a public resource served by the Companies House
-    API.
+  <p class="govuk-body">The Companies House API requires authentication credentials to be sent with
+    each request, which take the form of an API key.</p>
+  <p class="govuk-body">To add an API key:
   </p>
+  <ol class="list list-number govuk-body">
+    <li>Go to <a href="https://www.example.com/">Application overview</a></li>
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/authentication.html]: I think there is an Overview page, please link to it.

File: src/main/resources/templates/dev-hub/authentication.html [hunk 2]
```diff
@@ -10,57 +10,48 @@
...
-    application. This will allocate a unique key to the
-    application which can be sent with any
-    <code>GET</code>
-    request for a public resource served by the Companies House
-    API.
+  <p class="govuk-body">The Companies House API requires authentication credentials to be sent with
+    each request, which take the form of an API key.</p>
+  <p class="govuk-body">To add an API key:
   </p>
+  <ol class="list list-number govuk-body">
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/authentication.html]: Shouldn't this be a `govuk-list` not `gov-uk-body`?

File: src/main/resources/templates/dev-hub/authentication.html [hunk 3]
```diff
@@ -10,57 +10,48 @@
...
-      href="http://en.wikipedia.org/wiki/Basic_access_authentication" class="govuk-link">HTTP
-    Basic Authentication</a> to transmit an API key between the
-    client application and the server. Basic authentication is
-    usually made up of a username and password pair; the
-    Companies House API takes the username as the API key and
-    ignores the password, so can be left blank.
+    The Companies House API uses HTTP basic access authentication to send an API key between the
+    client application and the server.
+
+
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/authentication.html]: Remove redundant blank lines - they will be collapsed by HTML to a space.

Should these be separate `<p>` paragraph tags?

File: src/main/resources/templates/dev-hub/authentication.html [hunk 4]
```diff
@@ -10,57 +10,48 @@
...
-                class="string">/company/00000006</span> HTTP/1.1</span>
-<span class="attribute">Host</span>: <span class="string">api.companieshouse.gov.uk</span>
-<span class="attribute">Authorization</span>: <span class="string">Basic bXlfYXBpX2tleTo=</span>
-            </code>
-        </pre>
+  <pre><code>curl -XGET -u my_api_key: https:<span>//api.companieshouse.gov.uk/company/00000006</span></code></pre>
+  <pre><code class="http">
+  <span class="request">GET <span class="string">/company/00000006</span> HTTP/1.1</span>
+  <span class="attribute">Host</span>: <span class="string">api.companieshouse.gov.uk</span>
+  <span class="attribute">Authorization</span>: <span class="string">Basic bXlfYXBpX2tleTo=</span>
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/authentication.html]: Indentation seems wrong here.

File: src/main/resources/templates/dev-hub/developerGuidelines.html [hunk 5]
```diff
@@ -11,61 +11,99 @@
...
+    <p class="govuk-body">
+      You can make up to 600 requests within a five-minute period. If you exceed this limit, you
+      will receive a <code>429 Too Many Requests</code> HTTP status code for each request made
+      within the
+      remainder of the five-minute window. At the end of the period, your rate limit will reset back
+      to its maximum value of 600 requests.
+    </p>
+    <p class="govuk-body">
+      If you have an application that requires a higher rate limit than this default, <a
+        th:href="@{{chsUrl}/help/contact-us(chsUrl=${@environment.getProperty('chs.url')})}">contact
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/developerGuidelines.html]: Does this `chsUrl `go to the main domain service with a relevant contact page?

File: src/main/resources/templates/dev-hub/developerGuidelines.html [hunk 6]
```diff
@@ -11,61 +11,99 @@
...
-    <h2 class="govuk-heading-m">Securing API keys</h2>
-    <p class="govuk-body">It is important to keep your API keys secure, to prevent them being
-      discovered, your
-      account being compromised and your rate-limit quota being exceeded. There are a few best
-      practices that can help with this:</p>
+  <section id="keySecurity">
+    <h2 class="govuk-heading-m">API key security </h2>
+    <p class="govuk-body">It is important to keep your API keys secure. This will prevent them from
+      being discovered, your account from being compromised and your rate-limit quota from being
+      exceeded. </p>
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/developerGuidelines.html]: Paragraphs seem to have a space character before they are closed. Please clean that up.

File: src/main/resources/templates/dev-hub/getStarted.html [hunk 7]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/getStarted.html]: Remove this blank line if possible.

File: src/main/resources/templates/dev-hub/getStarted.html [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+  <section id="register">
+    <h3 class="govuk-heading-m">Register a Companies House user account </h3>
+    <p class="govuk-body">The Companies House API
+      <a class="govuk-link" href="https://www.example.com/">overview</a>
+      describes the basic operation of the API.
+
+      You must <a class="govuk-link"
+                  href="/signin">register a user account</a> with Companies House to explore and
+      perform tests with the Companies House API.
+
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/getStarted.html]: Should this be 1 paragraph or 3?

File: src/main/resources/templates/dev-hub/getStarted.html [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+      Being RESTful, each record of information is assigned a globally unique URI, and the
+      operations available on each resource are directly mapped to HTTP verbs. Data can be easily
+      consumed by issuing simple <code>GET</code> requests on the required resource URI.
+
+
+      Read the <a class="govuk-link"
+                  th:href="@{{manageApplicationsUrl}/(manageApplicationsUrl=${@environment.getProperty('manageApplications.url')})}">Companies
+        House REST API overview</a> for more information about our API approach, REST
+      web services and the JSON data format.
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/getStarted.html]: How many paragraphs is this supposed to be? 
Please remove the redundant blank lines.

File: src/main/resources/templates/dev-hub/getStarted.html [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+    </p>
+  </section>
+  <section id="authentication">
+    <h3 class="govuk-heading-m">API key authentication</h3>
+    <p class="govuk-body">Access to API services requires authentication. The Companies House API
+      requires authentication credentials to be sent with each request, which take the form of an
+      API key.
+
+      Learn how to <a class="govuk-link"
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/getStarted.html]: How many paragraphs?

File: src/test/java/uk/gov/ch/developer/docs/ApplicationVariables.java [hunk 11]
```diff
@@ -3,8 +3,8 @@
 public interface ApplicationVariables {
 
     String HOME_PATH = "/dev-hub";
-    String HOME_VIEW = "dev-hub/home";
-    String GETTINGSTARTED_PATH = "/dev-hub/getting-started";
-    String GETTINGSTARTED_VIEW = "dev-hub/gettingStarted";
-    String BADREQUEST_PATH = "bad-path";
+    String HOME_VIEW = "dev-hub/overview";
+    String GETSTARTED_PATH = "/dev-hub/getstarted";
+    String GETSTARTED_VIEW = "dev-hub/getStarted";
```
Comment (ch-dclarke) [inline src/test/java/uk/gov/ch/developer/docs/ApplicationVariables.java]: Please refactor to include underscores between each variable name for consistency.

Why is this an interface and not a final class?

File: src/main/resources/templates/dev-hub/authentication.html (hunk 1 above)
Comment (CH-JonathanBlake) [reply src/main/resources/templates/dev-hub/authentication.html]: I think this is referring to the logged in page given the context that it then goes on to create it.

File: src/main/resources/templates/dev-hub/authentication.html (hunk 3 above)
Comment (CH-JonathanBlake) [reply src/main/resources/templates/dev-hub/authentication.html]: Sure

File: src/main/resources/templates/dev-hub/authentication.html (hunk 4 above)
Comment (CH-JonathanBlake) [reply src/main/resources/templates/dev-hub/authentication.html]: Pre's are nasty.

File: src/main/resources/templates/dev-hub/developerGuidelines.html (hunk 5 above)
Comment (CH-JonathanBlake) [reply src/main/resources/templates/dev-hub/developerGuidelines.html]: This was copied from the footer bar which currently works in live.

File: src/main/resources/templates/dev-hub/authentication.html (hunk 2 above)
Comment (CH-JonathanBlake) [reply src/main/resources/templates/dev-hub/authentication.html]: Changed to govuk-list govuk-list--number

File: src/main/resources/templates/dev-hub/getStarted.html (hunk 8 above)
Comment (CH-JonathanBlake) [reply src/main/resources/templates/dev-hub/getStarted.html]: 2

File: src/main/resources/templates/dev-hub/getStarted.html (hunk 9 above)
Comment (CH-JonathanBlake) [reply src/main/resources/templates/dev-hub/getStarted.html]: 3

File: src/test/java/uk/gov/ch/developer/docs/ApplicationVariables.java (hunk 11 above)
Comment (CH-JonathanBlake) [reply src/test/java/uk/gov/ch/developer/docs/ApplicationVariables.java]: Because we implemented it that way in the first place. Do not remember.

**Pull request merged**